docs/scos/dev/architecture/module-api/definition-of-module-api.md

```diff
@@ -22,48 +22,49 @@ redirect_from:
   - /v2/docs/en/definition-api
   - /v1/docs/definition-api
   - /v1/docs/en/definition-api
+  - /docs/scos/dev/architecture/module-api/definition-of-module-api.html
 ---
 
-According to [Semantic Versioning](http://semver.org/), Spryker releases a major version “when there are incompatible API changes”. To make a sound decision about the version type, we need to define our internal APIs.
+According to [Semantic Versioning](http://semver.org/), we release a major version of a module when there are backward compatibility(BC) breaking changes in the Public API. This document declares what public and private APIs are.
 
-In the Spryker Commerce OS’s core, all public methods in theses locatable classes are considered as API:
+## Public API
 
-* [Facades](/docs/scos/dev/back-end-development/zed/business-layer/facade/facade.html)
-* [Clients](/docs/scos/dev/back-end-development/client/client.html)
-* [Query Containers](/docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.html)
-* [Services](/docs/scos/dev/back-end-development/messages-and-errors/registering-a-new-service.html)
-* Plugin defining protected methods in dependency providers.
+In the Spryker Commerce OS’s core, the following is the public API:
 
-And the interfaces which are implemented everywhere are also part of the API:
+* Public methods in these locatable classes:
+    * [Facades](/docs/scos/dev/back-end-development/zed/business-layer/facade/facade.html)
+    * [Clients](/docs/scos/dev/back-end-development/client/client.html)
+    * [Query Containers](/docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.html)
+    * [Services](/docs/scos/dev/back-end-development/messages-and-errors/registering-a-new-service.html)
+    * Plugin defining protected methods in dependency providers.
```

Comment:
The section is "Public methods"
but this example is a private method. 
So we can move it to other section

Reply:
we describe private as the rest, so I will just remove it.